tools/commands.py

```diff
@@ -14,18 +14,24 @@
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
 
-from tools.logging.logging_util import get_logger
 import os
 import sys
 import asyncio
+import signal
+from threading import Thread
+from concurrent.futures import CancelledError
 
+from tools.logging.logging_util import get_logger
 from backtesting.collector.data_collector import DataCollector
 from config.config import encrypt
 from tools.tentacle_creator.tentacle_creator import TentacleCreator
 from tools.tentacle_manager.tentacle_manager import TentacleManager
 
 
 class Commands:
+
+    BOT = None
```

Comment:
I think this is not a bad idea, but we need to uniform this with other part of the program.

For example, for interfaces we added a `bot_instance`  in __init__.py and a function `get_bot()`.

 Do you think we can do the same here ?

Reply:
Absolutely, much better like this